scripts/dpf-sanity-checks.sh

```
@@ -0,0 +1,346 @@
+
+echo -e "\noc get nodes --kubeconfig=${hosted_kubecfg} output:"
+oc get nodes --kubeconfig=${hosted_kubecfg}
+
+echo -e "\noc get pods -n dpf-operator-system --kubeconfig=${admin_kubecfg} -o wide output:"
+oc get pods -n dpf-operator-system --kubeconfig="${admin_kubecfg}" -o wide
+
+# Find doca-hb-pod names on each DPU worker node
+echo -e "\noc get pods -n dpf-operator-system --kubeconfig=${hosted_kubecfg} -o wide output:"
+oc get pods -n dpf-operator-system --kubeconfig="${hosted_kubecfg}" -o wide
+
+doca_hbn_pod_worker_node1=$(oc get pods -n dpf-operator-system --kubeconfig="${hosted_kubecfg}" -o wide | grep "${worker_node1}" | grep hbn | awk {'print $1'})
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)